tests/webapp/api/test_performance_data_api.py

```diff
@@ -235,3 +236,115 @@ def test_filter_data_by_framework(webapp, test_repository, test_perf_signature,
     assert len(datums) == 1
     assert datums[0]['job_id'] == 1
     assert datums[0]['signature_id'] == 2
+
+
+def test_filter_signatures_by_interval(webapp, test_repository, test_perf_signature,
+                                        test_perf_signature_same_hash_different_framework):
```

Comment:
The `test_perf_signature_same_hash_different_framework` fixture isn't necessary here (or below)